### PR TITLE
Add flow annotations to components/utils folder

### PR DESCRIPTION
**Pull request by bhargav175**
**What**:
Annotate /components/utils folder with flow-type

**Why**:
To get started with flow annotations in the website.

**How**:
Added type annotations for function args, return values and variable declarations.

### CODE REVIEW

File: components/utils/md-to-html.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ import stripIndent from './strip-indent'
```
Comment (kentcdodds) [inline components/utils/md-to-html.js]: Do we need `// @flow` at the top of this file?

File: components/utils/strip-indent.js [hunk 2]
```diff
@@ -1,17 +1,23 @@
+//@flow
```
Comment (kentcdodds) [inline components/utils/strip-indent.js]: Can we add a space here so it's `// @flow`?

File: components/utils/md-to-html.js (hunk 1 above)
Comment (bhargav175) [inline components/utils/md-to-html.js]: Yes, Looks like I forgot to add that one. Will add it.

File: components/utils/strip-indent.js (hunk 2 above)
Comment (bhargav175) [inline components/utils/strip-indent.js]: Yes. Will do.

**Comment by kentcdodds** [comment]
Cool! Thanks!

**Pull request merged**